numpy/distutils/mingw32ccompiler.py

```diff
@@ -465,7 +464,7 @@ def _build_import_library_amd64():
 
     # generate import library from this symbol list
     cmd = ['dlltool', '-d', def_file, '-l', out_file]
-    subprocess.Popen(cmd)
+    return subprocess.call(cmd)
```

Comment:
Why the `return` here?

Perhaps would make more sense as
```suggestion
    subprocess.check_call(cmd)
```

Reply:
Yeah, not needed I think, seems it is used in some other places where it is not needed, so probably why Matti it got in there. As a side note, the test looked a bit funny, but fine to me.